packages/flutter_client_sdk/pubspec.yaml

```diff
@@ -12,11 +12,11 @@ environment:
 dependencies:
   flutter:
     sdk: flutter
-  package_info_plus: ">=4.2.0 <6.0.0"
-  device_info_plus: ^9.1.1
```

Comment:
It looks like only connectivity plus introduced changes? So this one could be ">=9.1.1 <11.0.0"?

Reply:
Yes, good idea!